src/includes/performance/automatic-instrumentation-integrations/dotnet.mdx

```diff
@@ -1,3 +1,27 @@
+## DiagnosticSource Integration
+
+Starting with version 3.9.0, the SDK will automatically integrate with Entity Framework Core and SQLClient whenever available. Those integrations will automatically be activated if your project matches one of the following conditions:
+* Includes `Sentry.AspNet` 3.9.0 or higher
+* Includes `Sentry.AspNetCore` 3.9.0 or higher
+* Targets .NET Core 3.0 or higher (for example .NET 5)
+
+Even if your project doesn't meet any of those requirements, you'll be able to manually activate those instrumentations by including the package 'Sentry.DiagnosticSource' and enabling it during on the SDK's initialization.
```

Comment:
Do you mean for Sentry.DiagnosticSource to be in code style or in quotation marks? If you want quotation marks, use double quotes rather than single ones. I've edited it to be in code style, because I wasn't sure which you meant.

```suggestion
Even if your project doesn't meet any of those requirements, you can manually activate those instrumentations by including the package `Sentry.DiagnosticSource` and enabling it during on the SDK's initialization.
```